src/connect.c

```diff
@@ -826,7 +825,11 @@ void
 stl_update_connects_remove_1(stl_file *stl, int facet_num) {
   int j;
 
-  if (stl->error) return;
+  if (
+    stl->error ||
+    facet_num < 0
```

Comment:
What does `facet_num < 0` indicate here? Is it an invalid input that should be indicated by setting `stl->error`?

Reply:
It is only an argument value check which does not have to indicate any syntactical error. The file provided in the issue is actually syntactically correct but does not make much sense (it is a line). So this fix just makes sure we don't do any invalid 
(nonsense) operations and just skip this particular procedure.